### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

CI should pass.
Use the GUI, it should work as expected.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8652/

File: package.json [hunk 1]
```diff
@@ -15,58 +15,58 @@
...
+    "react-dnd-html5-backend": "^2.6.0",
+    "react-dom": "^16.3.2",
     "shapeup": "^0.2.5",
     "xterm": "^2.9.2",
     "yui": "3.11.0",
     "zip": "^1.2.0"
   },
   "devDependencies": {
-    "babel-cli": "6.24.1",
+    "babel-cli": "^6.26.0",
```
Comment (huwshimi) [inline package.json]: All these updates with `^` are now non-frozen versions. Is that OK?

Comment (hatched) [reply package.json]: Yeah because we lock it in the package-lock.json so I didn't feel it necessary to remove them.

Comment (huwshimi) [reply package.json]: Sure thing!

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**